app/controllers/home_controller.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
 class HomeController < ApplicationController
+  # before_action :authenticate_user!
```

Comment:
Don't we need this? so it prevent non logged in user from accessing the home screen 🤔

Reply:
Ahh, yes your right, I commented it before when I was playing around with Devise 😅 Updated in  https://github.com/aetheryna/google-scraper-ruby/pull/31/commits/a3ec449edaed5d16a742c05e2ffbe5d02f27275b